### ISSUE TITLE
ipdf - docs are out of date and incomplete (Trac #1298)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1292]
rst docs are light, and barely gloss over IPDF usage. links are dead.

doxygen docs are incomplete (several "Comming soon"'s)
doxygen docs are out of date (refer to the old Makefile build system, "plan to use the icetray unit system")

Migrated from https://code.icecube.wisc.edu/ticket/1298
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "rst docs are light, and barely gloss over IPDF usage. links are dead.\n\ndoxygen docs are incomplete (several \"Comming soon\"'s)\ndoxygen docs are out of date (refer to the old Makefile build system, \"plan to use the icetray unit system\")", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo simulation", 
    "summary": "ipdf - docs are out of date and incomplete", 
    "priority": "blocker", 
    "keywords": "documentation", 
    "time": "2015-08-28T20:21:09", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T04:57:45: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "claudio.kopper" to "olivas"
* olivas changed priority from "normal" to "blocker"
* olivas changed milestone from "" to "IceRec V5"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:57:46: kjmeagher changed owner from "olivas" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20151124T16:23:31: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "accepted" to "closed"
* kjmeagher commented:

> In changeset 139760:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139760"
> clean up and unify documentation. fixes #1292
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
